example-fileupload/README.md

```diff
@@ -0,0 +1,507 @@
+ - `src/components/FileUploader.vue` This is what handles file uploads
+
+Let's get started then
+
+## `src/App.vue`
+App.vue will be how we wrap things up, but we need to start creating the basics before we work on the others, including the refresh files from server code and basic styling.
+
+### The Template section
+
+We'll start by creating the basic template of this Web App, in the `src/App.vue` file go ahead and replace the `<template>` section with the following.
+```html
+<template>
+  <div class="FileCore">
+    <h1>Appwrite File Upload</h1>
+    <!-- File Uploader will go here -->
+    <h2>Uploaded Files:</h2>
+    <div v-for="file in allFiles" :key="file.$id">
+      <!-- File Component will go here -->
+    </div>
+  </div>
+</template>
+```
+Breaking down this template it's mainly HTML apart from the `v-for` section where we loop through the `allFiles` variable which we will create in the `data()` function later.
+
+### The Script section
+Next, we will create the data state, `created()` function (to run the refreshFiles on launch) and `getAllFiles()` method that will be used to get a list of the latest files on from the server.
+
+First, we will add the Appwrite SDK we made earlier by placing the following at the very top, even before the export default:
+```js
+import { appwrite } from './utils'
+```
+
+Next, we will create the data() which will store the variables (state) which we will use in `App.vue`, we're only going to use one variable as the rest is handled by the other components. Go ahead and add the following within the `export default` object.
+
+```js
+data() {
+    return {
+        allFiles: []
+    }
+},
+```
+We'll also create the `getAllFiles()` method now, Create a new object in the `export default` object called `methods` and add the following code within it:
+```js
+getAllFiles: function() {
+    let promise = appwrite.storage.listFiles(); // Get all files from server
+
+    promise.then((response) => { // wait until we get a response
+        console.log(response); // Success
+        this.allFiles = response.files // if success then set the allFiles variable to the files returned from server
+    }, function(error) {
+        console.error(error); // If we fail then print the error to the console.
+    });
+} 
+```
+This function does the following:
+
+ 1. Requests all the files from the server
+ 2. When the request is processed from the server if it's successful then set the allFiles variable to the file list returned
+ 3. If we fail then log the error into the console for further inspection and debugging.
+
+and finally, we will create a `created()` function, in VueJS the `created()` function is always executed when the component is created hence its name, we use this to trigger the initial get of data when the `App.vue` component is first created. Go ahead and add this code into the `export default` object and make sure it isn't in the `methods`object or else it will not execute
```

Comment:
`and` should start with an uppercase letter. I'm also missing an sentence break after `it will not execute`